go.mod

```diff
@@ -5,6 +5,7 @@ go 1.19
 require (
 	github.com/bufbuild/connect-go v1.5.1
 	github.com/klauspost/connect-compress v1.0.0
+	github.com/planetscale/vitess-types v0.0.0-20230210165505-c0e8b7520c81
```

Comment:
Just making sure don't you want to reference your submodule?

Reply:
Correct, the submodule is intentionally only the actual `.proto` files since they're needed to compile them. But for Go code, as a module, we'd use the actual module as a normal dependency for runtime.